tests/unit/reconcile/conftest.py

```diff
@@ -105,3 +105,157 @@ def schema():
 @pytest.fixture
 def expr():
     return parse_one("SELECT col1 FROM DUAL")
+
+
+@pytest.fixture
+def snowflake_databricks_schema():
```

Comment:
move to the individual test_schema_compare file only.

Reply:
and also you can simplify this using a small function

Reply:
Done. Following the same approach for now.